src/main/java/com/epam/ta/reportportal/ws/controller/LaunchController.java

```diff
@@ -255,64 +263,59 @@ public LaunchResource getLaunch(@PathVariable String projectName, @PathVariable
 		return getLaunchMessageHandler.getLaunchNames(ProjectUtils.extractProjectDetails(user, projectName), value);
 	}
 
-	//	@RequestMapping(value = "/compare", method = GET)
-	//	@ResponseBody
-	//	@ResponseStatus(OK)
-	//	@ApiOperation("Compare launches")
-	//	public Map<String, List<ChartObject>> compareLaunches(@PathVariable String projectName, @RequestParam(value = "ids") Long[] ids,
-	//			@AuthenticationPrincipal ReportPortalUser user) {
-	//		return getLaunchMessageHandler.getLaunchesComparisonInfo(normalizeId(projectName), ids);
-	//	}
+	@GetMapping(value = "/compare")
+	@ResponseBody
+	@ResponseStatus(OK)
+	@ApiOperation("Compare launches")
+	public List<ComparisonStatisticsContent> compareLaunches(@PathVariable String projectName, @RequestParam(value = "ids") Long[] ids,
+			@AuthenticationPrincipal ReportPortalUser user) {
+		return getLaunchMessageHandler.getLaunchesComparisonInfo(normalizeId(projectName), ids);
```

Comment:
instead of noramlizeId(projectName) please use ProjectUtils.extractProjectDetails